test/testutils/TimeUtils.scala

```diff
@@ -0,0 +1,14 @@
+package testutils
+
+import java.time.{ZoneId, ZonedDateTime}
+
+trait TimeUtils {
+  def withSystemTime[T](hour: Int, minute: Int, zoneId: Option[ZoneId] = None)(
```

Comment:
Be worth checkign there are any tests that need the timezone (especially after looking at the one I commented on above).

Arguably, if we want to make the server time independent of all the logic we shoulkd pick a random timezone and make sure that all the tests still pass!

Reply:
We now have a test that checks the duration method works in any arbitrary timezone.